src/main/java/com/flansmod/client/model/ModelVehicle.java

```diff
@@ -60,12 +74,23 @@ public void render(DriveableType type)
 		renderPart(drillHeadModel);
 		for(ModelRendererTurbo[] mods : ammoModel)
 			renderPart(mods);
+		for(ModelRendererTurbo[] latm : leftAnimTrackModel)
```

Comment:
So in the driveable creation box, it renders all the tracks at once? Surely you just want the first frame?